lib/admin/WP_Auth0_Admin_Features.php

```diff
@@ -378,21 +373,28 @@ public function render_override_wp_avatars( $args = array() ) {
 	}
 
 	public function basic_validation( $old_options, $input ) {
-		// SLO will be turned off in WP_Auth0_Admin_Features::sso_validation() if SSO is not on.
-		$input['singlelogout']        = ( isset( $input['singlelogout'] ) ? $input['singlelogout'] : 0 );
-		$input['override_wp_avatars'] = ( isset( $input['override_wp_avatars'] ) ? $input['override_wp_avatars'] : 0 );
+		$input['sso'] = ! empty( $input['sso'] ) ? 1 : 0;
+
+		// Turn SLO off if SSO is off.
+		$input['singlelogout']        = ! empty( $input['singlelogout'] ) && $input['sso'] ? 1 : 0;
```

Comment:
Using `empty` here because that array key may not be set. Not using `empty` for `$input['sso']` because that is always normalized above.